include/vcpkg/base/messages.h

```diff
@@ -919,6 +943,10 @@ namespace vcpkg
                     (msg::actual),
                     "{actual} is the provided format string",
                     "invalid format string: {actual}");
+    DECLARE_MESSAGE(InvalidPackageFeature,
+                    (msg::package_name, msg::value),
+                    "'{value}' is a port feature.",
+                    "Port {package_name} does not have a {value} feature");
```

Comment:
As I said in the other one, I think this should be merged with `FailedToFindPortFeature`.